client/js/models/preview.js

```diff
@@ -17,9 +17,11 @@ module.exports = {
       return data
     },
     openPanel: (data, state) => {
+      document.body.classList.add('panel-open') // not ideal
```

Comment:
Maybe this would be better in the page view file? We could do something like `${state.preview.isPanelOpen : set ? unset  class}`.

I have no idea. Obviously not _supposed_ to do this.